scale/scheduler/models.py

```diff
@@ -10,27 +12,36 @@
 
 
 class SchedulerManager(models.Manager):
-    '''Provides additional methods for handling scheduler db entry
-    '''
+    """Provides additional methods for handling scheduler db entry
+    """
 
     def get_master(self):
-        '''Gets the current master scheduler instance for the cluster.
+        """Gets the current master scheduler instance for the cluster.
 
         :returns: The master scheduler.
         :rtype: :class:`scheduler.models.Scheduler`
-        '''
+        """
         try:
             return Scheduler.objects.get(pk=1)
         except Scheduler.DoesNotExist:
             logger.exception('Initial database import missing master scheduler: 1')
             raise
 
+    def initialize_scheduler(self):
```

Comment:
The initialization code is nice because it only runs once. When the fixture runs, it overwrites existing model values. So whenever we do a Scale deploy, the scheduler is always automatically unpaused, which is not ideal or expected for a user.

I think we should use "run once" initialization code for database fields that act as configuration that needs to be set up once, but then should honor the user's settings (Scheduler model) and use fixtures for models that need to be updated as Scale is in order to function properly (Job Type models, etc).